app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -499,5 +773,20 @@ class RecentlyPlayedFragmentTest {
       throw UnsupportedOperationException()
     }
   }
-}
 
+  /** Custom class to check number of spans occupied by an item at a given position. */
```

Comment:
Can we have this class in Utility folder of androidTest.

Reply:
Done.

Reply:
LGTM.